### PR TITLE
Fixed a bug that causes false-alarm warning log message when loading job configs

### PR DESCRIPTION
**Pull request by liyinan926**
Signed-off-by: Yinan Li <liyinan926@gmail.com>

### CODE REVIEW

**Comment by liyinan926** [comment]
@zliu41 can you review this?

File: gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java [hunk 1]
```diff
@@ -226,18 +236,15 @@ private static void loadJobConfigsRecursive(List<Properties> jobConfigs, Propert
...
-    return Sets.newHashSet(Iterables.transform(jobConfigFileExtensionsIterable, new Function<String, String>() {
-          @Override
-          public String apply(String input) {
-            if (input == null) {
-              throw new NullPointerException("Given input is null: " + input);
-            }
-            return input.toLowerCase();
-          }
-        }));
+    return ImmutableSet.copyOf(Iterables.transform(jobConfigFileExtensionsIterable, new Function<String, String>() {
```
Comment (zliu41) [inline gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java]: Is it better to use case insensitive set here?

Comment (liyinan926) [inline gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java]: Yeah, we could use a case insensitive `TreeSet`, but to me that is more for sorting.

**Comment by zliu41** [comment]
LGTM

**Pull request merged**